FieldworkAIDriver.lua

```diff
@@ -288,9 +288,13 @@ end
 
 function FieldworkAIDriver:stop(msgReference)
 	self:stopWork()
-	-- persist last fieldwork waypoint data (for 'start at current waypoint')
-	self.aiDriverData.lastFieldworkCourseHash = self.fieldworkCourse:getHash()
-	self.aiDriverData.lastFieldworkWaypointIx = self.fieldworkCourse:getCurrentWaypointIx()
+	-- Issue with persisting fieldwork data in multiplayer, as the client doesn't maintain the fieldworkcourse (client never calls start)
+	-- Persisted fieldwork data isn't needed by the client anyway, as its all handled by the server (also in start)
+	if not CpManager.isMP or not courseplay.isClient then
```

Comment:
wouldn't just g_server ~= nil suffice here? That seems to be used by the Giants code all over the place.

Reply:
I went with that as I thought it was easier to read, but probably better to be consistent. Updated it.